File: Apps/Java/clientJava/src/main/java/com/mycompany/projeto/artefato/sprint/entidades/Empresa.java
package com.mycompany.projeto.artefato.sprint.entidades;

public class Empresa {
    private String id_empresa;
    private String nome;
    private String pais;
    private String cidade;
    private Integer fk__staff;

    public Empresa(String id_empresa, String nome, String pais, String cidade,
            Integer fk__staff) {
        this.id_empresa = id_empresa;
        this.nome = nome;
        this.pais = pais;
        this.cidade = cidade;
        this.fk__staff = fk__staff;
    }
}

File: website/models/empresa.js
'use strict';

module.exports = (sequelize, DataTypes) => {
    let empresa = sequelize.define('empresa',{	
		id: {
			field: 'id_empresa',
			type: DataTypes.STRING,
			primaryKey: true,
			autoIncrement: false
		},	
		nome: {
			field: 'nome',
			type: DataTypes.STRING,
			allowNull: false
		},
		pais: {
			field: 'pais',
			type: DataTypes.STRING,
			allowNull: false
		},
		cidade: {
			field: 'cidade',
			type: DataTypes.STRING, 
			allowNull: false
		},
        fk_staff: {
			field: 'fk_staff',
			type: DataTypes.INTEGER, 
			allowNull: false
		},
	}, 
	{
		tableName: 'empresa', 
		freezeTableName: true, 
		underscored: true,
		timestamps: false,
	});

    return empresa;
};
